src/completers/git.zsh

```diff
@@ -554,6 +554,69 @@ _fzf_complete_git() {
         return
     fi
 
+    if [[ $subcommand = stash ]]; then
+        local prefix_option completing_option
+        local git_options_argument_required=(
+            --pathspec-from-file
+            -m
+            --message
+        )
+
+        if completing_option=$(_fzf_complete_parse_completing_option "$prefix" "$last_argument" "${(F)git_options_argument_required}" ''); then
+            if [[ $completing_option = --* ]]; then
+                prefix_option=$completing_option=
+            else
+                prefix_option=${prefix%%${completing_option[-1]}*}${completing_option[-1]}
+            fi
+            prefix=${prefix#$prefix_option}
+        fi
+
+        local stash_subcommand=${arguments[${arguments[(i)$subcommand]} + 1]}
+        case $stash_subcommand in
+            show)
```

Comment:
`git stash show` can take the options that are used in `git diff`. This is parted from `apply`, `drop` and `pop` for when it is implemented.